### PR TITLE
Load community and event editors with state

### PR DESCRIPTION
**Pull request by lpil**
circlecircle

- [x] Move community editor one level up
- [x] State load from server re-initializes forms
- [x]  Community editor is loaded with community state
- [x] Event editor loaded with state (picking from correct event)

### CODE REVIEW

File: app/components/CommunityEditor/index.js [hunk 1]
```diff
@@ -21,26 +21,27 @@ const Form = () => (
...
     <h2>Preview</h2>
     <div className="preview">
-      <CommunityPreview />
+      <Community {...communityProps} />
     </div>
   </div>
 );
 
 CommunityEditor.propTypes = {
+  communityProps: PropTypes.shape(Community.propTypes).isRequired,
```
Comment (AndrewBestbier) [inline app/components/CommunityEditor/index.js]: Nice! Why haven't i done this before...

File: app/sagas/load-state/index.js [hunk 2]
```diff
@@ -1,13 +1,30 @@
...
+  });
+  return eventActions;
+}
+
 export function* loadStateWorker() {
   try {
     const res = yield call(api.fetchSiteState);
     yield put(siteStateLoaded(res.data));
+    yield put(formInitialize('community', res.data.community));
+    yield put(formInitialize('conference', res.data.conference));
```
Comment (zoeabryant) [inline app/sagas/load-state/index.js]: can we have the form names as constants?

File: app/components/CommunityEditor/index.js (hunk 1 above)
Comment (lpil) [inline app/components/CommunityEditor/index.js]: Innit. :D

File: app/sagas/load-state/index.js (hunk 2 above)
Comment (lpil) [inline app/sagas/load-state/index.js]: Yess!

File: app/sagas/load-state/index.js [hunk 3]
```diff
@@ -1,13 +1,30 @@
...
+import { eventIDToFormName } from '../../names/event';
 import * as api from '../../api';
 import {
   siteStateLoaded, siteStateLoadFailed,
 } from '../../actions/persistence';
 
+
+function buildEventActions(events) {
+  const eventActions = [];
+  _forOwn(events, (value, key) => {
```
Comment (AndrewBestbier) [inline app/sagas/load-state/index.js]: May be a bit overboard to require lodash as a dependency.  Could do the following if you agree:
`
Object.keys(obj).forEach(function(key,index) {});
`

Comment (lpil) [inline app/sagas/load-state/index.js]: This only imports the one function, seems cleaner than polyfilling Object.keys

Comment (AndrewBestbier) [inline app/sagas/load-state/index.js]: We will probably use lodash plenty in the future of this project too so I guess it doesn't really matter

**Pull request merged**